src/extremeweatherbench/evaluate.py

```diff
@@ -194,25 +192,112 @@ def _evaluate_case(
             join="inner",
         )
         for data_var in individual_case.data_vars:
-            case_results[data_var] = {}
+            case_results["gridded"][data_var] = {}
             forecast_da = spatiotemporal_subset_ds[data_var].compute()
             gridded_obs_da = time_subset_gridded_obs_ds[data_var].compute()
             for metric in individual_case.metrics_list:
                 metric_instance = metric()
                 logging.debug(
-                    "metric %s computing for %s", metric_instance.name, data_var
+                    "gridded metric %s computing for %s", metric_instance.name, data_var
                 )
                 result = metric_instance.compute(forecast_da, gridded_obs_da)
-                case_results[data_var][metric_instance.name] = result
+                case_results["gridded"][data_var][metric_instance.name] = result
+                logger.debug(
+                    "gridded, %s, %s, %s", data_var, metric_instance.name, result
+                )
     if point_obs is not None:
-        raise NotImplementedError("Point obs evaluation not implemented as of 0.1.0")
+        case_results["point"] = {}
+        case_subset_point_obs = point_obs.loc[point_obs["id"] == individual_case.id]
+        case_subset_point_obs = case_subset_point_obs.rename(columns=utils.ISD_MAPPING)
+        case_subset_point_obs["longitude"] = utils.convert_longitude_to_360(
+            case_subset_point_obs["longitude"]
+        )
+        case_subset_point_obs = utils.unit_check(case_subset_point_obs)
+        case_subset_point_obs = case_subset_point_obs[
+            (
+                case_subset_point_obs["latitude"]
+                >= spatiotemporal_subset_forecast_ds["latitude"].min().values
+            )
+            & (
+                case_subset_point_obs["latitude"]
+                <= spatiotemporal_subset_forecast_ds["latitude"].max().values
+            )
+            & (
+                case_subset_point_obs["longitude"]
+                >= spatiotemporal_subset_forecast_ds["longitude"].min().values
+            )
+            & (
+                case_subset_point_obs["longitude"]
+                <= spatiotemporal_subset_forecast_ds["longitude"].max().values
+            )
+        ]
```

Comment:
This should probably be a helper function with independent testing (lots of "gotchas" can arise on the edges of this sort of interval selection).